phpBB/language/en/acp/extensions.php

```diff
@@ -130,4 +150,32 @@
 
 	'META_FIELD_NOT_SET'	=> 'Required meta field %s has not been set.',
 	'META_FIELD_INVALID'	=> 'Meta field %s is invalid.',
+
+	'EXTENSIONS_CATALOG_SETTINGS'	=> 'Extensions catalog settings',
+	'ENABLE_ON_INSTALL'				=> 'Enable extensions when installing',
+	'PURGE_ON_REMOVE'				=> 'Purge extensions when removing',
+	'ENABLE_PACKAGIST'				=> 'Enable Packagist usage',
+	'ENABLE_PACKAGIST_EXPLAIN'		=> 'Use packagist to browse extensions? Beware that packagist may contain extensions not validated by phpBB.',
+	'ENABLE_PACKAGIST_CONFIRM'		=> 'Are you sure you want to enable packagist?',
+	'COMPOSER_REPOSITORIES'			=> 'Repositories',
+	'COMPOSER_REPOSITORIES_EXPLAIN'	=> 'List of composer repositories (must be the base url of the packages.json file), one per line.',
```

Comment:
"Add URLs to Composer repositories of phpBB extensions to search here, one per line (must be the base url of the packages.json file)."